userguide/dailyuse/anbox.rst

```diff
@@ -14,25 +14,26 @@ Supported devices
 
 Make sure your device is supported:
 
-- Meizu Pro 5 (*turbo*)
-- BQ M10 HD (*cooler*)
-- BQ M10 FHD (*frieza*)
+Meizu Pro 5 (codename: *turbo*, name of the boot partition: *bootimg*)
+BQ M10 HD (codename: *cooler*, name of the boot partition: *boot*)
+BQ M10 FHD (codename: *frieza*, name of the boot partition: *boot*)
 
-You will need the device codename (the italic part in the brackets) for the installation.
+You will need the device codename and the name of your boot partition for the installation.
 
 How to install
 --------------
 
 .. warning::
     Because this feature is in such an early stage of development, the installation is only recommended for experienced users.
 
-- Install the 16.04/devel channel on your supported device
-- Open a terminal and run ``export CODENAME="turbo"``, but replace the part between the quotes with your devices codename. See the above list.
-- Activate developer mode on your device, connect it to your computer and run the following commands::
+- :doc:`Install <../install>` the 16.04/devel channel on your supported device
+- Open a terminal and run ``export CODENAME="turbo" && export PARTITIONNAME="bootimg"``, but replace the part between the quotes respectively with the codename and name of the boot partition for your device. See the above list.
+- Activate developer mode on your device.
+- Connect the device to your computer computer and run the following commands::
 
     adb shell sudo reboot -f bootloader
     wget http://cdimage.ubports.com/anbox-images/anbox-boot-$CODENAME.img
-    sudo fastboot flash boot anbox-boot-$CODENAME.img
+    sudo fastboot $PARTITIONNAME boot anbox-boot-$CODENAME.img
```

Comment:
It'll be `fastboot flash *partition* <image.img>`

Reply:
oh shit, good catch